core/src/main/java/com/example/AutomatedReviewTest.java

```diff
@@ -0,0 +1,53 @@
+package com.example;
+
+import java.util.*;
+import java.util.logging.*;
+
+/**
+ * The goal of this code is to find out if code analysis will automatically add
+ * commentary to a pull request when it finds problems with the code.
+ */
+public final class AutomatedReviewTest {
+    
+    private Logger LOGGER = Logger.getLogger(AutomatedReviewTest.class.getName());
+    
+    protected double one   = 1.0d;
+    protected double two   = 2.0d;
+    protected double three = 5.0d;
+    protected double four  = 4.0d;
+    
+    private AutomatedReviewTest() {
+        canBeStatic();
+    }
+    
+    /**
+     * Method description without description of parameters or return value.
+     */
+    public boolean hello() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Document this method return value. ([squid:UndocumentedApi](https://sonarcloud.io/organizations/coderanch-corral/rules#rule_key=squid%3AUndocumentedApi))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.coderanch%3Ablackjack&pullRequest=9&issues=AWZEEZasvOM6zLh9bdFn&open=AWZEEZasvOM6zLh9bdFn)

[ISSUE_KEY]: AWZEEZasvOM6zLh9bdFn